docs/guides/key-manager/upgrade-lsp6.md

```diff
@@ -187,6 +341,61 @@ await upgradeLSP6();
 
 <!-- prettier-ignore-end -->
 
+  </TabItem>
+
+  <TabItem value="ethersjs" label="ethers.js">
+
+<!-- prettier-ignore-start -->
+
+```js title="Imports & Constants"
+import UniversalProfile from '@lukso/lsp-smart-contracts/artifacts/UniversalProfile.json';
+import LSP6KeyManager from '@lukso/lsp-smart-contracts/artifacts/LSP6KeyManager.json';
+import { ethers } from 'ethers';
+
+const provider = new ethers.JsonRpcProvider('https://rpc.l16.lukso.network');
+
+const privateKey = '0x...';
+const universalProfileAddress = '0x...';
+const keyManagerAddress = '0x...';
+
+const upgradeLSP6 = async () => {
+  // Initialize the controller account
+  const account = new ethers.Wallet(privateKey).connect(provider);
+
+  // Initialize your current LSP6 Key Manager
+  const oldKeyManager = new ethers.Contract(keyManagerAddress, LSP6KeyManager.abi);
+
+  // Deploy a new LSP6 Key Manager
+  const NewKeyManager = new ethers.ContractFactory(
+    LSP6KeyManager.abi,
+    LSP6KeyManager.bytecode,
+  );
+  const newKeyManager = await NewKeyManager.deploy(universalProfileAddress);
```

Comment:
maybe this is a bit confusing, can we call them something else like `const newKeyManager = await keyManagerContract.deploy` or something?